style.css

```diff
@@ -24,8 +24,9 @@
     padding-left: 20px;
     position: fixed;
     width: 250px;
-    background-color: aquamarine;
+    background-color: rgb(85, 110, 235);
```

Comment:
You can use variables here , use it wherever possible , for font color also , 
Rest looks fine , you can merge